airbyte-integrations/connectors/source-iterable/source_iterable/source.py

```diff
@@ -2,20 +2,14 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 #
 
-from typing import Any, List, Mapping, Tuple
+from typing import Any, List, Mapping
 
-import requests.exceptions
-from airbyte_cdk.models import SyncMode
-from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.declarative.yaml_declarative_source import YamlDeclarativeSource
 from airbyte_cdk.sources.streams import Stream
 from airbyte_cdk.sources.streams.http.requests_native_auth import TokenAuthenticator
-from source_iterable.utils import read_full_refresh
 
 from .streams import (
-    AccessCheck,
```

Comment:
Should this stream be present in the manifest?

Reply:
No, we don't need this stream any more. Instead this stream we use `list` stream to check connection.

Reply:
👍 Can you update the PR description with that info, and any other major changes?